src/Credential.jsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import "./Credential.css";
+
+class Credential extends React.Component {
```

Comment:
Maybe `LoginForm` or some such might be more apt?

Reply:
Sounds nicer! Let me update them